taggers/place_tagger.py

```
@@ -0,0 +1,432 @@
+        except Exception as e:
+            logger.warning(f"Could not load {label} data: {e}")
+
+    def process_item(self, item, img) -> List[Tag]:
+        tags = []
+
+        if self.args.force and getattr(item, "tags", None):
+            for t in item.tags:
+                if t.name in (TAG_NAME, TAGGER_TAG_NAME) and t.id is not None:
+                    try:
+                        self.service.delete_item_tag(self.dataset_id, item.id, t.id)
+                    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)